### PR TITLE
Ui Fixes for issue #15,#17 in Jira ML-260

### PR DESCRIPTION
**Pull request by fazlan-nazeem**
please note that there has been a formatting issue before, and it has been also fixed with this PR.

### CODE REVIEW

File: apps/ml/site/home/home.jag [hunk 1]
```diff
@@ -168,12 +155,12 @@ $( document ).ready(function() {
             xhr.setRequestHeader("Authorization", "Basic " + authEncoded);
         },
         success : function(response){
-        	$("#dataset-count").html(response.length);
+            $("#dataset-count").html(response.length);
         },
         error :  function(response){
-        	//TODO: handle errors
+            //TODO: handle errors
```
Comment (cdathuraliya) [inline apps/ml/site/home/home.jag:161]: Add common error handling.

Comment (fazlan-nazeem) [inline apps/ml/site/home/home.jag:161]: Fixed with PR https://github.com/wso2/carbon-ml/pull/200

File: apps/ml/site/project/projects.jag [hunk 2]
```diff
@@ -668,6 +667,19 @@ include("../../includes/tenantAware.jag");
         }
     }
 
+    function appendNavigation(projectName){
+          // need for navigation
+        var queryParam = getParameterByName("projectName");
+
+        if(queryParam.length > 0 && projectName == queryParam){
```
Comment (prabathabey) [inline apps/ml/site/project/projects.jag:674]: Formatting errors

File: apps/ml/site/project/projects.jag [hunk 3]
```diff
@@ -668,6 +667,19 @@ include("../../includes/tenantAware.jag");
         }
     }
 
+    function appendNavigation(projectName){
```
Comment (prabathabey) [inline apps/ml/site/project/projects.jag:670]: Formatting issues.

File: apps/ml/site/project/projects.jag [hunk 4]
```diff
@@ -668,6 +667,19 @@ include("../../includes/tenantAware.jag");
...
 
+    function appendNavigation(projectName){
+          // need for navigation
+        var queryParam = getParameterByName("projectName");
+
+        if(queryParam.length > 0 && projectName == queryParam){
+            $("#collapse0").removeClass('in');
+              return 'in';
+        }     
+        else{
```
Comment (prabathabey) [inline apps/ml/site/project/projects.jag:678]: More formatting issues.

File: apps/ml/site/project/projects.jag (hunk 3 above)
Comment (fazlan-nazeem) [inline apps/ml/site/project/projects.jag:670]: Fixed with PR https://github.com/wso2/carbon-ml/pull/201

File: apps/ml/site/project/projects.jag (hunk 2 above)
Comment (fazlan-nazeem) [inline apps/ml/site/project/projects.jag:674]: Fixed with https://github.com/wso2/carbon-ml/pull/201

File: apps/ml/site/project/projects.jag (hunk 4 above)
Comment (fazlan-nazeem) [inline apps/ml/site/project/projects.jag:678]: Fixed with https://github.com/wso2/carbon-ml/pull/201

File: apps/ml/site/analysis/view-model.jag [hunk 5]
```diff
@@ -339,6 +339,15 @@ $( document ).ready(function() {
           }
     });
 
+    if( fromCompare == 'true'){
+         $("#nav-analysis-anchor").attr('href','../project/compare.jag?projectId=' + projectId + '&projectName=' + projectName + '&dataseId=' + datasetId);
+    }
+    else{
```
Comment (Kishanthan) [inline apps/ml/site/analysis/view-model.jag:345]: Formatting issue. "else" statement should start right after the "if" closing curly brackets.

File: apps/ml/site/analysis/view-model.jag [hunk 6]
```diff
@@ -339,6 +339,15 @@ $( document ).ready(function() {
           }
     });
 
+    if( fromCompare == 'true'){
```
Comment (Kishanthan) [inline apps/ml/site/analysis/view-model.jag:342]: Formatting issue.

Comment (fazlan-nazeem) [inline apps/ml/site/analysis/view-model.jag:342]: Fixed with PR https://github.com/wso2/carbon-ml/pull/202

File: apps/ml/site/analysis/view-model.jag (hunk 5 above)
Comment (fazlan-nazeem) [inline apps/ml/site/analysis/view-model.jag:345]: fixed with PR https://github.com/wso2/carbon-ml/pull/202

**Pull request merged**